modules/core/src/test/scala/org/scalasteward/core/nurture/UpdateInfoUrlFinderTest.scala

```diff
@@ -157,18 +161,7 @@ class UpdateInfoUrlFinderTest extends CatsEffectSuite with Http4sDsl[MockEff] {
     )
 
     assertEquals(
-      possibleVersionDiffs(
-        GitHub,
-        onPremForgeUrl,
-        uri"https://scalacenter.github.io/scalafix/",
-        v1,
-        v2
-      ),
-      List.empty
-    )
```

Comment:
This assertion is deleted because it was testing that `possibleVersionDiffs()` would return an empty list if called with a repo url that doesn't have an identifiable `ForgeType` - `possibleVersionDiffs()` is now only ever called if a `ForgeType` has been identified, and that information is part of the required `ForgeRepo` parameter on the `possibleVersionDiffs()` method.

Consequently this test no longer applies for the new method signature/parameter semantic!  A similar assertion was deleted for `possibleUpdateInfoUrls()` below.

Reply:
Makes sense! Thanks for cleaning that up 😊